include/swift/IDE/SyntaxModel.h

```diff
@@ -146,6 +147,7 @@ struct SyntaxStructureNode {
     case SyntaxStructureKind::StaticVariable:
     case SyntaxStructureKind::ClassVariable:
     case SyntaxStructureKind::Parameter:
+    case SyntaxStructureKind::Subscript:
```

Comment:
This accessor (`isVariable()`) is only being used in one place in the Syntax Model and only for things that have substructure that needs to be walked.  Would it be OK to just rename it `hasSubstructure()` or something?